app/controllers/user_rule_effects_controller.rb

```diff
@@ -0,0 +1,79 @@
+class UserRuleEffectsController < ApplicationController
+  def index
+    authorize workflow
+    effects = policy_scope(UserRuleEffect).where(user_rule_id: params[:user_rule_id])
+    render json: effects
+  end
+
+  def show
+    authorize workflow
+    @effect = policy_scope(UserRuleEffect).find(params[:id]) or not found
```

Comment:
Huh? What's `or not found` here? Where is the `found` method defined?

Reply:
this is almost certainly supposed to be `not_found` but that just leads to your next point :)